index.html

```diff
@@ -208,14 +219,18 @@ <h4 class="link-title">Skills</h4>
             </section>
         </article>
     </section>
-    <section class="contact-form">
-        <h3 class="about-me-headline">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-        <form class="contact-me-form" action="https://formspree.io/f/xgedpkeq" method="post">
-            <input id="full-name-input" type="text" name="full-name" placeholder="Full name" required maxlength="30">
-            <input id="email-input" type="email" name="email" pattern="+@globex\.com" placeholder="Email address" minlength="6" required>
-            <textarea id="textarea-input" name="message" maxlength="500" placeholder="Enter text here" required></textarea> 
-            <button class="get-my-resume button-input highlight" type="submit">Get in touch</button>
-        </form>
+    <section class="get-in-touch-last">
+        <section class="contact-form">
+            <h3 class="about-me-headline">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+            <form class="contact-me-form" action="https://formspree.io/f/xgedpkeq" method="post">
+                <input id="full-name-input" type="text" name="full-name" placeholder="Full name" required maxlength="30">
+                <input id="email-input" type="email" name="email" pattern="+@globex\.com" placeholder="Email address" minlength="6" required>

```

Comment:
- [ ] There should be two input fields for first name and last name. kindly implement it in the desktop version.
- [ ] Kindly implement the spacing of text and form as the form takes more spacing than the content.
## Figma design
![figma](https://user-images.githubusercontent.com/81727988/165779578-61e83dab-f21d-463a-bded-b025908f7d3a.png)

Reply:
Hi @roshan-bajgain ,

Thank you for the heads up. The instruction had said we can use only one output of the full name for both version as below.

![image](https://user-images.githubusercontent.com/59999191/165787436-31a5582c-6f8f-4f5c-b531-6225b0bf48e1.png)

Reply:
I don't seem to understand what I need to implement by the following statement. May you please elaborate?

"Kindly implement the spacing of text and form as the form takes more spacing than the content."

Reply:
@Draxeytina, I am sorry for this review you can ignore it. Thanks

Reply:
@Draxeytina If you see the contact form then the form took more space than the content so kindly give more width to the form container so that It looks wider than the content.